StorageAccessAPI/explainer.md

```diff
@@ -1,4 +1,11 @@
 # Storage Access API Explainer
+
+## Status of this Document
+This document is intended as a starting point for engaging the community and standards bodies in developing collaborative solutions fit for standardization. As the solutions to problems described in this document progress along the standards-track, we will retain this document as an archive and use this section to keep the community up-to-date with the most current standards venue and content location of future work and discussions.
+* This document status: **Active**
```

Comment:
Would it be relevant to also link to the Github location the spec is being discussed and iterated on atm: https://github.com/privacycg/storage-access/

Reply:
Hmm. If the standards conversation has completely moved to the Privacy CG, then maybe this explainer should be marked as Archived? Otherwise, can you elaborate on the plan for this explainer in relation to the other work? This new SOTD is supposed to redirect new folks to the place where the latest standards discussions are happening. If there's worthwhile feedback we're still looking to have in MicrosoftEdge/MSEdgeExplainers relevant to this document, then we can indicate both venues...

Reply:
I think this is still relevant w.r.t our position on the API and for our Chromium impl efforts. However how that the discussion is ongoing in the Privacy CG we should probably link it too. Having both seems reasonable at this point in time.

Reply:
Thanks!